hw/xbox/mcpx_apu.c

```diff
@@ -1424,6 +1453,7 @@ static void process_voice(MCPXAPUState *d,
       voice_get_mask(d, v, NV_PAVS_VOICE_CFG_FMT, NV_PAVS_VOICE_CFG_FMT_V6BIN),
       voice_get_mask(d, v, NV_PAVS_VOICE_CFG_FMT, NV_PAVS_VOICE_CFG_FMT_V7BIN)
     };
+#endif
```

Comment:
Why is this being disabled? As far as I remember, this was confirmed against hardware.

Reply:
This was disabled as I disabled mixing to bins below. It should be re-enabled however before this gets merged